[0.000000] (-) TimerEvent: {}
[0.000497] (-) JobUnselected: {'identifier': 'mr2_realsense'}
[0.000609] (-) JobUnselected: {'identifier': 'yolobot_recognition'}
[0.000734] (-) JobUnselected: {'identifier': 'yolov8_msgs'}
[0.000848] (buffalo_robot) JobQueued: {'identifier': 'buffalo_robot', 'dependencies': OrderedDict()}
[0.002269] (buffalo_robot) JobStarted: {'identifier': 'buffalo_robot'}
[0.099791] (-) TimerEvent: {}
[0.201509] (-) TimerEvent: {}
[0.302193] (-) TimerEvent: {}
[0.402858] (-) TimerEvent: {}
[0.503653] (-) TimerEvent: {}
[0.604432] (-) TimerEvent: {}
[0.705229] (-) TimerEvent: {}
[0.806469] (-) TimerEvent: {}
[0.907825] (-) TimerEvent: {}
[1.008505] (-) TimerEvent: {}
[1.084639] (buffalo_robot) StderrLine: {'line': b'Traceback (most recent call last):\n  File "/usr/lib/python3/dist-packages/colcon_parallel_executor/executor/parallel.py", line 75, in execute\n    loop.run_until_complete(future)\n  File "/usr/lib/python3.8/asyncio/base_events.py", line 603, in run_until_complete\n    self.run_forever()\n  File "/usr/lib/python3.8/asyncio/base_events.py", line 570, in run_forever\n    self._run_once()\n  File "/usr/lib/python3.8/asyncio/base_events.py", line 1823, in _run_once\n    event_list = self._selector.select(timeout)\n  File "/usr/lib/python3.8/selectors.py", line 468, in select\n    fd_event_list = self._selector.poll(timeout, max_ev)\nKeyboardInterrupt\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File "/usr/lib/python3/dist-packages/colcon_core/executor/__init__.py", line 91, in __call__\n    rc = await self.task(*args, **kwargs)\n  File "/usr/lib/python3/dist-packages/colcon_core/task/__init__.py", line 93, in __call__\n    return await task_method(*args, **kwargs)\n  File "/usr/lib/python3/dist-packages/colcon_ros/task/ament_python/build.py", line 102, in build\n    return await extension.build(additional_hooks=additional_hooks)\n  File "/usr/lib/python3/dist-packages/colcon_core/task/python/build.py", line 90, in build\n    available_commands = await self._get_available_commands(\n  File "/usr/lib/python3/dist-packages/colcon_core/task/python/build.py", line 188, in _get_available_commands\n    output = await check_output(\n  File "/usr/lib/python3/dist-packages/colcon_core/subprocess.py", line 129, in check_output\n    assert not rc, f\'Expected {args} to pass: {stderr_data}\'\nAssertionError: Expected [\'/usr/bin/python3\', \'-W\', \'ignore:setup.py install is deprecated\', \'setup.py\', \'--help-commands\'] to pass: Traceback (most recent call last):\n  File "setup.py", line 1, in <module>\n    from setuptools import setup\n  File "/home/jsagx-6/.local/lib/python3.8/site-packages/setuptools/__init__.py", line 14, in <module>\n    from . import version as _version_module\n  File "/home/jsagx-6/.local/lib/python3.8/site-packages/setuptools/version.py", line 1, in <module>\n    from ._importlib import metadata\n  File "/home/jsagx-6/.local/lib/python3.8/site-packages/setuptools/_importlib.py", line 43, in <module>\n    disable_importlib_metadata_finder(metadata)\n  File "/home/jsagx-6/.local/lib/python3.8/site-packages/setuptools/_importlib.py", line 12, in disable_importlib_metadata_finder\n    import importlib_metadata\n  File "/usr/lib/python3/dist-packages/importlib_metadata/__init__.py", line 591, in <module>\n    __version__ = version(__name__)\n  File "/usr/lib/python3/dist-packages/importlib_metadata/__init__.py", line 553, in version\n    return distribution(distribution_name).version\n  File "/usr/lib/python3/dist-packages/importlib_metadata/__init__.py", line 526, in distribution\n    return Distribution.from_name(distribution_name)\n  File "/usr/lib/python3/dist-packages/importlib_metadata/__init__.py", line 190, in from_name\n    dist = next(dists, None)\n  File "/home/jsagx-6/.local/lib/python3.8/site-packages/setuptools/_vendor/importlib_metadata/__init__.py", line 739, in <genexpr>\n    path.search(prepared) for path in map(FastPath, paths)\n  File "/home/jsagx-6/.local/lib/python3.8/site-packages/setuptools/_vendor/importlib_metadata/__init__.py", line 631, in search\n    return self.lookup(self.mtime).search(name)\n  File "/home/jsagx-6/.local/lib/python3.8/site-packages/setuptools/_vendor/importlib_metadata/_functools.py", line 80, in wrapper\n    return cached_method(*args, **kwargs)\n  File "/home/jsagx-6/.local/lib/python3.8/site-packages/setuptools/_vendor/importlib_metadata/__init__.py", line 641, in lookup\n    return Lookup(self)\n  File "/home/jsagx-6/.local/lib/python3.8/site-packages/setuptools/_vendor/importlib_metadata/__init__.py", line 655, in __init__\n    name = low.rpartition(".")[0].partition("-")[0]\nKeyboardInterrupt\n\n'}
[1.087253] (buffalo_robot) JobEnded: {'identifier': 'buffalo_robot', 'rc': 'SIGINT'}
[1.095877] (-) EventReactorShutdown: {}
